structopt-derive/src/attrs.rs

```diff
@@ -291,6 +296,39 @@ impl Attrs {
 
                 VerbatimDocComment(ident) => self.verbatim_doc_comment = Some(ident),
 
+                DefaultValue(ident, lit) => {
+                    let val = if let Some(lit) = lit {
+                        quote!(#lit)
+                    } else {
+                        let ty = if let Some(ty) = self.ty.as_ref() {
+                            ty
+                        } else {
+                            abort!(
+                                ident.span(),
+                                "#[structopt(default_value)] (without an argument) can be used \
+                                only on field level";
+
+                                note = "see \
+                                    https://docs.rs/structopt/0.3.5/structopt/#magical-methods")
+                        };
+
+                        let static_name = format_ident!("__STRUCTOPT_DEFAULT_VALUE_{}", fresh_id());
```

Comment:
Do you really need this Mame generation? Can't you simply rely on scope for the name of the lazy_static?

Reply:
Hmm, I thought the scoping rules are not applicable for statics for some reason. No idea why I thought that, it just ended up in my brain somehow :) I guess I spend too much time with C.